src/pupil_labs/invisible_lsl_relay/outlets.py

```diff
@@ -10,7 +9,7 @@
     pi_gaze_channels,
 )
 
-VERSION = "1.0"
+VERSION = "2.1.0"
```

Comment:
The version should no longer be hardcoded. Use `from pupil_labs.invisible_lsl_relay import __version__` instead